### DISCUSSION TITLE
OpenVPN Version

### DISCUSSION THREAD

**Discussion by rudythebear** [pivpn/pivpn#1686] [Q&A] [answered]
Hi,

Firstly - a word of thanks - this has made setting up a VPN on my RPi very, very easy - it's a dream!

Now a question - versions and updates - it looks like the version of OpenVPN is 2.5.1 (found by doing > openvpn --version), but it looks like the latest version of OpenVPN server is 2.11.3.

have I got the wrong end of the stick and is 2.5.1 the version of PiVPN?

Cheers.

**Comment by TinCanTech** [comment]
You are confusing the FOSS OpenVPN 2.5.1 with the closed source server product from OpenVPN.Inc.

PiVPN only uses the Free version.

**Reply by rudythebear** [reply]
thank you for the speedy clarification!

**Comment by coolapso** [comment, marked as answer]
Let's clarify ... 
PiVPN is an installer, 
PiVPN installs and configures openvpn or wireguard for you. 
Currently PiVPN (Installer script / scripts ) is on Version is 4.1.5: https://github.com/pivpn/pivpn/releases/tag/v4.1.5 
The Protocol version installed by PiVPN is the one provided by the distribution repositories, and in your case OpenVPN 2.5.1

as @TinCanTech mentions, OpenVPN.inc also has "enterprise" product based on the  OpenVPN protocol called OpenVPN access server, which has another version which is 2.11.3. 

So 3 different "softwares" with 3 different versions and moving at different paces.

**Reply by rudythebear** [reply]
lovely - thanks for the clarification! 

"So 3 different "softwares" with 3 different versions and moving at different paces." hehhe! makes perfect sense! :)
